### PR TITLE
docs: storage

### PR DESCRIPTION
**Pull request by alexgao001**
## Description

storage docs

## Rationale

tell us why we need these changes...

### CODE REVIEW

File: docs/api/blockchain-cli/storage-provider.md [hunk 1]
```diff
@@ -5,32 +5,171 @@ order: 4
...
+
+```sh
+gnfd query sp storage-provider 1 --node https://greenfield-chain.bnbchain.org:443
+```
+
+## Query a Storage Provider by address
 
 ```shell
-$ bash ./deployment/localup/localup.sh all 3 7
+gnfd query sp storage-provider-by-operator-address [sp-id] [flags]
```
Comment (KeefeL) [inline docs/api/blockchain-cli/storage-provider.md:61]: sp-id -> sp-address?

File: docs/api/blockchain-cli/storage-provider.md [hunk 2]
```diff
@@ -5,32 +5,171 @@ order: 4
...
+      "gc_address": "0x0a1C8982C619B93bA7100411Fc58382306ab431b",
+      "maintenance_address": "0xbE03316B1D7c3FCB69136e47e02442d6Fb3396dB",
+      "endpoint": "https://sp0.greenfield.io",
+      "deposit": {
+        "denom": "BNB",
+        "amount": "500000000000000000000"
+      },
+      "read_price": "0.108",
+      "store_price": "0.016",
+      "free_read_quota": 1073741824,
```
Comment (KeefeL) [inline docs/api/blockchain-cli/storage-provider.md:119]: "read_price":"0.1469890427",
    "store_price":"0.02183945725",
    "free_read_quota": 1073741824,

File: docs/api/blockchain-cli/storage-provider.md (hunk 1 above)
Comment (alexgao001) [reply docs/api/blockchain-cli/storage-provider.md]: fixed

File: docs/api/blockchain-cli/storage-provider.md (hunk 2 above)
Comment (alexgao001) [reply docs/api/blockchain-cli/storage-provider.md:119]: modified

**Review by KeefeL — APPROVED** [review]

**Pull request merged**